packages/embedded/tsconfig.json

```diff
@@ -9,6 +9,9 @@
     "lib": ["ESNext"],
     "esModuleInterop": true,
     "outDir": "./dist",
+    "paths": {
+      "@/*": ["./src/*"]
```

Comment:
have you made sure this path alias works when published? i still see (in the compiled js) that the paths are "@/const", for instance

Reply:
removing this to reduce possible errors